src/Components/Catalog/Poster/Poster.js

```diff
@@ -1,33 +1,14 @@
 import style from '../../../styles/Poster.module.css'
+import Film from '../Film/Film';
 
 const Poster = ({films,loading}) => {
     if(loading){
         return <h2>Loading...</h2>
     }
 
-
     return(
         <ul className={style.list}>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
-            <li className={style.poster}>               
-                <img className={style.image} src = {films.Poster}/> 
-                <p className={style.title}>{films.Title}</p>
-                <p className={style.year}>{films.Year}</p>                         
-            </li>
+        <Film films={films} />
```

Comment:
you should rename this component as MovieList and pass to the <Film/> only one filme props. Also use here map/ not in Film component. Also all checks you should do here.

Reply:
done